infrastructure/ai/prompt_manager.py

```
@@ -22,10 +22,39 @@
 logger = logging.getLogger(__name__)
 
 # 内置种子 JSON 路径
+# 生产版(AITEXT_PROD_DATA_DIR): 从数据目录加载
+# 开发版: 从源码目录加载
 _DEFAULT_SEED_PATH = (
     Path(__file__).resolve().parent / "prompts" / "prompts_defaults.json"
 )
 
+
+def _get_seed_path() -> Path:
+    """获取种子文件路径，打包后优先从数据目录或 bundle 资源目录读取。
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)